core/domain/stats_services_test.py

```diff
@@ -151,6 +151,118 @@ def test_calls_to_stats_methods(self):
         self.assertEqual(stats_for_new_exploration_log.times_called, 1)
         self.assertEqual(stats_for_new_exp_version_log.times_called, 1)
 
+    def test_exploration_changes_effect_on_exp_issues_model(self):
+        """Test the effect of exploration changes on exploration issues model.
+        """
+        exp_id = 'exp_id'
+        test_exp_filepath = os.path.join(
+            feconf.TESTS_DATA_DIR, 'string_classifier_test.yaml')
+        yaml_content = utils.get_file_contents(test_exp_filepath)
+        assets_list = []
+
+        # Exploration is created, exploration issues model must also be created.
+        exp_services.save_new_exploration_from_yaml_and_assets(
+            feconf.SYSTEM_COMMITTER_ID, yaml_content, exp_id,
+            assets_list)
+        exploration = exp_services.get_exploration_by_id(exp_id)
+        exp_issues = stats_services.get_exp_issues(exp_id, exploration.version)
+        self.assertEqual(exp_issues.exp_version, 1)
```

Comment:
I know we always add 1 to the `exp_version`, but wouldn't it be "easier" to always compare it/set it to the value of the current exploration's version?

Reply:
In tests, we could compare it to the current exp version, but then explicitly mentioning it might be a nice idea too, because the reader has an idea of how version transcending is happening?

Reply:
Right. In my mind, any change in an exploration's versioning should be reflected exactly by a similar change to the playthrough's version, and so why not just make them use the same value?

That also makes it easy to check whether a playthrough applies to a specific version of an exploration.

Reply:
@brianrodri I don't really mind. Done.